src/ontology/uberon-edit.obo

```diff
@@ -223630,7 +223630,7 @@ def: "Blood located in the placenta." [http://orcid.org/0000-0001-6595-0902, htt
 intersection_of: UBERON:0000178 ! blood
 intersection_of: located_in UBERON:0001987 ! placenta
 relationship: dc-contributor http://orcid.org/0000-0002-1773-2692
-property_value: http://purl.org/dc/elements/1.1/date 2022-04-04T11:06:09Z xsd:dateTime
+property_value: dcterms-date 2022-04-04T11:06:09Z xsd:dateTime
```

Comment:
@matentzn - is this the way to do it for CL as well? I updated my Protégé preferences.

Reply:
If it is not, it should be!

Reply:
Will add to the next CL meeting agenda so everyone is on the same page (although it just may be myself who had this setting!).

Reply:
See https://oboacademy.github.io/obook/lesson/contributing-to-obo-ontologies/#setup under section "Setting Preferences for New entities metadata". See the screenshot. Should this be updated?

Reply:
I put it on the agenda for the workshop on Tuesday.